### PR TITLE
i18n: shorten the prompts

### PR DESCRIPTION
**Pull request by OriginCode**
Shorten the prompts to enhance the user experience of reading them.

### CODE REVIEW

File: i18n/en-US/systemd_boot_friend_rs.ftl [hunk 1]
```diff
@@ -20,38 +20,39 @@ create_folder = Creating folder structure for friend ...
...
+ask_overwrite = {$entry} already exists. Overwrite?
+ask_set_default = Set {$kernel} as default boot entry?
+select_install = Kernel(s) to register as boot entry(s)
+select_remove = Kernel(s) to remove from the boot menu
+select_default = Default kernel to boot from
+ask_init = Proceed with installing and initializing systemd-boot?
+ask_update = Proceed with searching and creating boot entries?
+ask_empty_bootarg = Generate the boot arguments?
+ask_current_bootarg = Use as default systemd-boot boot arguments?
+ask_current_root = Use as default systemd-boot boot arguments? (root={$root} rw)
```
Comment (liushuyu) [inline i18n/en-US/systemd_boot_friend_rs.ftl]:
```suggestion
ask_empty_bootarg = Auto generate the boot arguments?
ask_current_bootarg = Default boot arguments?
ask_current_root = Default boot arguments? (root={$root} rw)
```

File: i18n/en-US/systemd_boot_friend_rs.ftl [hunk 2]
```diff
@@ -20,38 +20,39 @@ create_folder = Creating folder structure for friend ...
...
-ask_empty_bootarg = Let systemd-boot-friend generate the boot arguments?
 prompt_current_bootarg = Detected current boot arguments (kernel command line):
-ask_current_bootarg = Use these as default systemd-boot boot arguments?
 prompt_current_root = Detected current root partition: {$root}
-ask_current_root = Use this for default systemd-boot boot arguments? (root={$root} rw)
-input_timeout = Please input the timeout (seconds) for systemd-boot to show the boot menu
\ No newline at end of file
+
+ask_overwrite = {$entry} already exists. Overwrite?
+ask_set_default = Set {$kernel} as default boot entry?
```
Comment (liushuyu) [inline i18n/en-US/systemd_boot_friend_rs.ftl]:
```suggestion
ask_set_default = Set {$kernel} as the default boot entry?
```

File: i18n/en-US/systemd_boot_friend_rs.ftl [hunk 3]
```diff
@@ -20,38 +20,39 @@ create_folder = Creating folder structure for friend ...
...
 prompt_init =
     systemd-boot-friend will now install and initialize systemd-boot, which will
     become the default EFI boot option on your system. If you already have GRUB or
     other bootloaders (such as Windows Boot Manager) installed, they will remain
     accessible from your EFI Boot Manager.
-ask_init = Proceed with installing and initializing systemd-boot?
 update = Updating boot entries ...
 prompt_update =
     Successfully initialized systemd-boot. Would you like systemd-boot-friend to
     search your {$src_path} directory for kernels and register them in systemd-boot
```
Comment (liushuyu) [inline i18n/en-US/systemd_boot_friend_rs.ftl]:
```suggestion
    search your `{$src_path}` directory for kernels and register them in systemd-boot
```

File: i18n/en-US/systemd_boot_friend_rs.ftl [hunk 4]
```diff
@@ -20,38 +20,39 @@ create_folder = Creating folder structure for friend ...
...
     configuration? If not, you could always do so by running
     `systemd-boot-friend update`.
```
Comment (liushuyu) [inline i18n/en-US/systemd_boot_friend_rs.ftl]: It's recommended to remove the "If not, you could always do so by running `systemd-boot-friend update`"  portion from this prompt.

Instead, add a logic to print this sentence when the user selected no.

```
[...] > no

You can add them later by running `systemd-boot-friend update`.
```

File: i18n/en-US/systemd_boot_friend_rs.ftl [hunk 5]
```diff
@@ -20,38 +20,39 @@ create_folder = Creating folder structure for friend ...
...
-    systemd-boot-friend detected an empty `BOOTARG=` field in your configuration.
-    This may cause system boot failures.
-ask_empty_bootarg = Let systemd-boot-friend generate the boot arguments?
-prompt_current_bootarg = Detected current boot arguments (kernel command line):
-ask_current_bootarg = Use these as default systemd-boot boot arguments?
-prompt_current_root = Detected current root partition: {$root}
-ask_current_root = Use this for default systemd-boot boot arguments? (root={$root} rw)
-input_timeout = Please input the timeout (seconds) for systemd-boot to show the boot menu
\ No newline at end of file
+ask_empty_bootarg = Auto generate the boot arguments?
```
Comment (MingcongBai) [inline i18n/en-US/systemd_boot_friend_rs.ftl]: Remove "the"

Comment (MingcongBai) [reply i18n/en-US/systemd_boot_friend_rs.ftl]: Replace "Auto" with "Automatically"

File: i18n/en-US/systemd_boot_friend_rs.ftl [hunk 6]
```diff
@@ -20,38 +20,39 @@ create_folder = Creating folder structure for friend ...
...
+ask_current_bootarg = Use the above as the default systemd-boot boot arguments?
```
Comment (MingcongBai) [inline i18n/en-US/systemd_boot_friend_rs.ftl]: Use the kernel parameters above as systemd-boot defaults?

File: i18n/en-US/systemd_boot_friend_rs.ftl [hunk 7]
```diff
@@ -20,38 +20,39 @@ create_folder = Creating folder structure for friend ...
...
+ask_current_root = Use `root={$root} rw` as the default systemd-boot boot arguments?
```
Comment (MingcongBai) [inline i18n/en-US/systemd_boot_friend_rs.ftl:57]:
```
Append `root={$root} rw` to the default systemd-boot kernel parameters?
```

File: i18n/en-US/systemd_boot_friend_rs.ftl (hunk 5 above)
Comment (MingcongBai) [reply i18n/en-US/systemd_boot_friend_rs.ftl]: Replace "boot arguments" with "kernel parameters"

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: i18n/en-US/systemd_boot_friend_rs.ftl (hunk 7 above)
Comment (OriginCode) [reply i18n/en-US/systemd_boot_friend_rs.ftl:57]: It's not append.

File: i18n/zh-CN/systemd_boot_friend_rs.ftl [hunk 8]
```diff
@@ -19,36 +19,37 @@ create_folder = 正在建立 friend 目录结构...
...
-select_default = 请选择要设为默认启动项的内核
 init = 正在安装并初始化 systemd-boot ...
-prompt_init =
-    systemd-boot-friend 即将安装及初始化 systemd-boot，并将其设置为默认 EFI 启动项。
-    完成后，您依旧可以在 EFI 启动管理器中访问其他已安装的启动引导器，如 GRUB 或
-    Windows 启动管理器。
-ask_init = 是否要安装并初始化 systemd-boot-friend？
+notice_init =
+    systemd-boot-friend 即将安装及初始化 systemd-boot，并将其设置为默认 EFI 启动项。完成
+    后，您依旧可以在 EFI 启动管理器中访问其他已安装的启动引导器，如 GRUB 或 Windows 启动
```
Comment (MingcongBai) [inline i18n/zh-CN/systemd_boot_friend_rs.ftl]: “您依旧可以从 EFI 启动管理器访问其他启动引导器，……”

File: i18n/zh-CN/systemd_boot_friend_rs.ftl [hunk 9]
```diff
@@ -19,36 +19,37 @@ create_folder = 正在建立 friend 目录结构...
...
+    管理器。
 update = 正在更新启动项 ...
-prompt_update =
-    systemd-boot 已成功初始化。是否要让 systemd-boot-friend 搜索 {$src_path} 中的内
-    核并将其登记至 systemd-boot 配置中？若不需要，您可以随时执行
-    `systemd-boot-friend update` 进行该操作。
-ask_update = 是否要安装所有内核并登记启动项？
-prompt_empty_bootarg =
+skip_update = 您可以在之后执行 `systemd-boot-friend update` 以加入启动项。
```
Comment (MingcongBai) [inline i18n/zh-CN/systemd_boot_friend_rs.ftl]: 您可以随时运行 `systemd-boot-friend update` 来登记启动项。

File: i18n/zh-CN/systemd_boot_friend_rs.ftl [hunk 10]
```diff
@@ -19,36 +19,37 @@ create_folder = 正在建立 friend 目录结构...
...
     动失败。
-ask_empty_bootarg = 需要 systemd-boot-friend 帮助您生成启动参数吗？
 prompt_current_bootarg = 检测到了当前使用的启动参数（内核命令行）：
-ask_current_bootarg = 是否要以此作为 systemd-boot 默认启动参数？
 prompt_current_root = 检测到了当前的根目录分区： {$root}
-ask_current_root = 是否要以此生成 systemd-boot 默认启动参数？（root={$root} rw）
-input_timeout = 请输入 systemd-boot 启动菜单的等待时长（秒）
\ No newline at end of file
+
+ask_overwrite = {$entry} 已存在。覆盖该文件？
```
Comment (MingcongBai) [inline i18n/zh-CN/systemd_boot_friend_rs.ftl]: {$entry} 已存在。是否覆盖该文件？

File: i18n/zh-CN/systemd_boot_friend_rs.ftl [hunk 11]
```diff
@@ -19,36 +19,37 @@ create_folder = 正在建立 friend 目录结构...
...
+ask_set_default = 将 {$kernel} 设为默认启动项？
```
Comment (MingcongBai) [inline i18n/zh-CN/systemd_boot_friend_rs.ftl]: 这类问题一律加“是否”开头，否则文法不通

File: i18n/zh-CN/systemd_boot_friend_rs.ftl [hunk 12]
```diff
@@ -19,36 +19,37 @@ create_folder = 正在建立 friend 目录结构...
...
+select_install = 要登记启动项的内核
+select_remove = 要从启动菜单移除的内核
+select_default = 默认启动的内核
```
Comment (MingcongBai) [inline i18n/zh-CN/systemd_boot_friend_rs.ftl]: 默认内核

File: i18n/zh-CN/systemd_boot_friend_rs.ftl [hunk 13]
```diff
@@ -19,36 +19,37 @@ create_folder = 正在建立 friend 目录结构...
...
+ask_init = 安装并初始化 systemd-boot-friend？
+prompt_update =
+    systemd-boot 已成功初始化。是否要让 systemd-boot-friend 搜索 `{$src_path}` 中的内核
+    并将其登记至 systemd-boot 配置中？
+ask_update = 安装所有内核并登记启动项？
+ask_empty_bootarg = 自动生成启动参数？
+ask_current_bootarg = 将上述启动参数设为 systemd-boot 默认参数？
+ask_current_root = 使用 `root={$root} rw` 生成 systemd-boot 默认启动参数？
+input_timeout = 启动菜单等待时长（秒）
```
Comment (MingcongBai) [inline i18n/zh-CN/systemd_boot_friend_rs.ftl]: “显示时长”会更好

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Comment by NeoChen1024** [comment]
zh_TW checked: OK

**Pull request merged**